editions/dev/tiddlers/ldorigo/High-Level Overview/Hooks/Hook: th-saving-tiddler.tid

```diff
@@ -1,10 +1,11 @@
 created: 20150908150314994
 modified: 20170209145506427
-tags: HookMechanism
+tags: Hooks
 title: Hook: th-saving-tiddler
+toc-order: 11
```

Comment:
To me it looks like this field will create a lot of maintenance (pain) in the future. It should be removed. We need to use the built in mechanism!